tools/wptrunner/wptrunner/browsers/webkit.py

```diff
@@ -69,6 +70,10 @@ def env_options():
     return {}
 
 
+def run_info_extras(**kwargs):
+    return {"wk_port": kwargs["webkit_port"]}
```

Comment:
Can we use `webkit_port` rather than `wk_port`?

Reply:
Yes.